### PR TITLE
fix media display when mp3

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

File: MediaAdminBundle/Transformer/MediaTransformer.php [hunk 1]
```diff
@@ -84,9 +84,13 @@ public function getName()
      */
     protected function generateMediaUrl($key)
     {
-        $route = $this->generateRoute('open_orchestra_media_get',
-            array('key' => $key),
-            UrlGeneratorInterface::ABSOLUTE_PATH);
-        return $this->mediaDomain . $route;
+        if($key == null) {
```
Comment (ngilain) [inline MediaAdminBundle/Transformer/MediaTransformer.php]: a space between 'if' and '(' would be nice :)

File: MediaAdminBundle/Transformer/MediaTransformer.php [hunk 2]
```diff
@@ -84,9 +84,13 @@ public function getName()
...
+            return '/bundles/openorchestramediaadmin/images/no_image_available.jpg';
```
Comment (ngilain) [inline MediaAdminBundle/Transformer/MediaTransformer.php]: You should use a constant for this string.

And maybe an application setting would be great :)

Comment (itkg-rboessel) [inline MediaAdminBundle/Transformer/MediaTransformer.php]: done @itkg-ngilain

**Comment by ngilain** [comment]
:+1:

**Comment by alavieille** [comment]
I do not agree to add the link directly, is possible to get the media by her id and used open_orchestra_media_get action?

**Comment by itkg-rboessel** [comment]
this image is not in gaufrette or database, it's in a bundle.

**Comment by ngilain** [comment]
Yes it's a design feature, not a media

**Comment by alavieille** [comment]
:+1:

**Pull request merged**